### PR TITLE
[@types/react] Add referrerPolicy to iframe attrs in React

### PR DESCRIPTION
**Pull request by charles-toller**
Please fill in this template.

- [ x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ x ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ x ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ x ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ x ] Provide a URL to documentation or source code which provides context for the suggested changes: 
[MDN showing referrerpolicy attribute for iframes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-referrerpolicy)
[React code allowing attribute `referrerPolicy`](https://github.com/aweary/react/blob/3b44102094f1d341097f2e4df9e899475afba7c3/src/renderers/dom/shared/HTMLDOMPropertyConfig.js#L124)
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/react/index.d.ts [hunk 1]
```diff
@@ -2546,7 +2547,7 @@ declare namespace React {
         hr: DetailedHTMLFactory<HTMLAttributes<HTMLHRElement>, HTMLHRElement>;
         html: DetailedHTMLFactory<HtmlHTMLAttributes<HTMLHtmlElement>, HTMLHtmlElement>;
         i: DetailedHTMLFactory<HTMLAttributes<HTMLElement>, HTMLElement>;
-        iframe: DetailedHTMLFactory<IframeHTMLAttributes<HTMLIFrameElement>, HTMLIFrameElement>;
+        iframe: DetailedHTMLFactory<IframeHTMLAttributes<HTMLElement>, HTMLIFrameElement>;
```
Comment (eps1lon) [inline types/react/index.d.ts]: Doesn't look like this change belongs here?

**Review by eps1lon — CHANGES_REQUESTED** [review]

File: types/react/index.d.ts (hunk 1 above)
Comment (charles-toller) [reply types/react/index.d.ts]: Removed.

**Review by ferdaber — APPROVED** [review]

**Comment by charles-toller** [comment]
@sandersn Sorry if you're the wrong person to ask, but you're the person I could find that merged a pull request with this issue. The Travis CI build is failing because it's timing out from the 500+ dependent packages, but the Azure Pipelines build passes. How do I get this merged if it was approved?

**Comment by sandersn** [comment]
You guessed right. I need to disable Travis but first I have to make typescript-bot understand Azure. Right now react is the only package with this problem so I've been manually merging them when requested.

**Pull request merged**